latest/ug/automode/auto-odcr.adoc

```diff
@@ -2,8 +2,14 @@ include::../attributes.txt[]
 
 [.topic]
 [#auto-odcr]
-= Control deployment of workloads into EC2 On-Demand Capacity Reservations with EKS Auto Mode
-:info_titleabbrev: Control ODCR Deployment
+= Control deployment of workloads into Capacity Reservations with EKS Auto Mode
+:info_titleabbrev: Control Reserved Capacity
+
+You can control the deployment of workloads onto reserved capacity. EKS Auto Mode supports EC2 On-Demand Capacity Reservations (ODCRs), and EC2 Capacity Blocks for ML. 
+
+TIP: By default, EKS Auto Mode automatically launches into open ODCRs. Once this feature is enabled, EKS Auto Mode will no longer automatically use open ODCRs.
```

Comment:
> By default, EKS Auto Mode automatically launches into open ODCRs

By default, EKS Auto Mode automatically launches into open ODCRs and ML Capacity Blocks?

> Once this feature is enabled

Suggest saying, "When using `capacityReservationSelectorTerms` in the NodeClass definition, EKS Auto Mode will no longer automatically use any open ODCRs"